src/operation.ts

```diff
@@ -48,7 +48,7 @@ export class GrpcOperation extends GrpcServiceObject {
    * longrunning.operation service.
    * @param {string} name - The operation name.
    */
-  constructor(parent: GrpcService|GrpcServiceObject, name: string) {
+  constructor(parent: GrpcService|GrpcServiceObject|{}, name: string) {
```

Comment:
why empty object here?

Reply:
in test at line 58 in "test/operation.ts" file test case is creating object of GrpcOperation with Fake Service object which can by any type as it is object , it can be handle by empty object or any type of typescript please provide suggestion - taken refrences from https://github.com/googleapis/nodejs-pubsub/pull/348/files